packages/paste-core/components/input-box/stories/inputbox.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import {storiesOf} from '@storybook/react';
+import {Box} from '@twilio-paste/box';
+import {Stack} from '@twilio-paste/stack';
+import {InputBox, InputBoxProps} from '../src';
+
+const Insert: React.FC = () => {
+  return <Box padding="space20" />;
+};
+
+const Input: React.FC = () => {
+  return <Box padding="space60" width="100%" />;
+};
+
+const CommonStories: React.FC<Omit<InputBoxProps, 'children'>> = ({variant, ...props}) => {
+  return (
+    <Stack orientation="vertical" spacing="space70">
+      <InputBox variant={variant} {...props}>
+        <Input />
+      </InputBox>
+      <InputBox variant={variant} type="hidden" {...props}>
+        <Input />
+      </InputBox>
+      <InputBox variant={variant} insertBefore={<Insert />} {...props}>
+        <Input />
+      </InputBox>
+      <InputBox variant={variant} insertAfter={<Insert />} {...props}>
+        <Input />
+      </InputBox>
+      <InputBox variant={variant} insertAfter={<Insert />} insertBefore={<Insert />} {...props}>
+        <Input />
+      </InputBox>
+    </Stack>
+  );
+};
+
+storiesOf('Components|Input box', module)
+  .add('default', () => {
+    return <CommonStories variant="default" />;
+  })
+  .add('readonly', () => {
+    return <CommonStories readOnly variant="default" />;
+  })
+  .add('disabled', () => {
+    return <CommonStories disabled variant="default" />;
+  })
+  .add('error', () => {
+    return <CommonStories hasError variant="default" />;
+  })
+  .add('inverse', () => {
+    return (
+      <Box padding="space40" backgroundColor="colorBackgroundBodyInverse">
+        <CommonStories variant="inverse" />
+      </Box>
```

Comment:
You could potentially do something similar to the inverse button stories for these. https://github.com/twilio-labs/paste/blob/main/packages/paste-core/components/button/stories/index.stories.tsx#L82

Not a blocker, just calling it out.